w3af/core/controllers/sql_tools/blind_sqli_time_delay.py

```diff
@@ -130,7 +130,7 @@ def is_injectable(self, mutant, delay_obj):
 
             v = Vuln.from_mutant('Blind SQL injection vulnerability', desc,
                                  severity.HIGH, response_ids, 'blind_sqli',
-                                 mutant)
+                                 mutant, vulndb_id=46)
```

Comment:
I believe there is not need to specify the `vulndb_id` parameter here. If the title specified in the `Vuln.from_mutant()`: `Blind SQL injection vulnerability` matches the title in the vulndb, then the information is associated automatically. That wasn't working? Why adding the explicit ID if it was?

Reply:
Yes, it was working. However it won't currently work for vulns with ID 10000 and higher. I'd opt to leave it as it is because it doesn't change anything at all for lower IDs and for higher IDs it allows to assign vulndb_id.